packages/angular/src/generators/application/lib/convert-to-standalone-app.ts

```diff
@@ -45,13 +45,11 @@ function updateMainEntrypoint(
 
 const standaloneComponentMainContents = (
   routerModuleSetup
-) => `import { enableProdMode${
-  routerModuleSetup ? `, importProvidersFrom` : ``
-} } from '@angular/core';
+) => `import { enableProdMode} from '@angular/core';
```

Comment:
nit:
```suggestion
) => `import { enableProdMode } from '@angular/core';
```
Formatting would make it right, but it's a change that caused the snapshot to change unnecessarily that we could easily avoid.